client/lib/setup/index.coffee

```diff
@@ -34,7 +34,7 @@ module.exports = (app) ->
     res.send 200, { nodejs: true }
 
   # Mount generic middleware & run setup modules
-  if 'production' is NODE_ENV
+  if 'production' is NODE_ENV or 'staging' is NODE_ENV
```

Comment:
Coffeescript pro tip: I like to use `in` for this kind of thing, e.g. `if NODE_ENV in ['production, 'staging']` (more helpful with a lot of `or` conditions.